admin-dev/themes/new-theme/js/pages/customer-groups/index.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import Grid from '@components/grid/grid';
+import BulkActionCheckboxExtension from '@components/grid/extension/bulk-action-checkbox-extension';
+import SubmitBulkExtension from '@components/grid/extension/submit-bulk-action-extension';
+import ExportToSqlManagerExtension from '@components/grid/extension/export-to-sql-manager-extension';
+import SortingExtension from '@components/grid/extension/sorting-extension';
+import FiltersResetExtension from '@components/grid/extension/filters-reset-extension';
+import ReloadListActionExtension from '@components/grid/extension/reload-list-extension';
+import ColumnTogglingExtension from '@components/grid/extension/column-toggling-extension';
+import SubmitRowActionExtension from '@components/grid/extension/action/row/submit-row-action-extension';
+import FiltersSubmitButtonEnablerExtension from '@components/grid/extension/filters-submit-button-enabler-extension';
+import LinkRowActionExtension from '@components/grid/extension/link-row-action-extension';
+
+const {$} = window;
+
+$(() => {
+  const customerGroups = new Grid('customer_groups');
+
+  customerGroups.addExtension(new BulkActionCheckboxExtension());
+  customerGroups.addExtension(new SubmitBulkExtension());
+  customerGroups.addExtension(new ExportToSqlManagerExtension());
+  customerGroups.addExtension(new SortingExtension())
```

Comment:
```suggestion
  customerGroups.addExtension(new SortingExtension());
```
Quick tip before you ask for review, or even before final push ideally You should run the different lint checks that we have:
- cs fixer `./vendor/bin/php-cs-fixer fix`
- PHP stan `php -d memory_limit=-1 ./vendor/bin/phpstan analyse -c phpstan.neon.dist`
- js linter `npm run lint` (in the appropriate theme source folder)
- scss linter `npm run scss-lint` (in the appropriate theme source folder)
(of course depending on which type of files you modified)

It will save you a lot of time in the long run